### PR TITLE
Correcting variables throughout the project

### PR DESCRIPTION
**Pull request by jimmynotjim**
We missed some of the variables when updating for v4, this corrects it.

## Changes

- Updated all the variables to the correct names, locations, and organization.
- Updated all the docs to reflect the correct variables.

## Testing

- Run `gulp build` and make sure it can compile the files.

## Review

- @Scotchester 
- @contolini 
- @cfarm 

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,9 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 -
 
 ### Changed
--
+- **all components:** Corrected the variables
+  - Updated all the variables to the correct names, locations, and organization
+  - Updated all the docs to reflect the correct variables
```
Comment (Scotchester) [inline CHANGELOG.md]: I feel like we had issues with nested changelog entries. Did we fix that?

File: src/cf-buttons/usage.md [hunk 2]
```diff
@@ -31,32 +31,43 @@ Overwrite them in your own project by duplicating the variable `@key: value`.
 [US Web Design Standards](https://github.com/18F/web-design-standards/blob/18f-pages/assets/_scss/core/_variables.scss)
```
Comment (Scotchester) [inline src/cf-buttons/usage.md]: This URL needs updating here. Also "U.S."

File: src/cf-forms/usage.md [hunk 3]
```diff
@@ -44,17 +44,45 @@ Overwrite them in your own project by duplicating the variable `@key: value`.
 [US Web Design Standards](https://github.com/18F/web-design-standards/blob/18f-pages/assets/_scss/core/_variables.scss)
```
Comment (Scotchester) [inline src/cf-forms/usage.md]: Update URL and "U.S."

File: src/cf-icons/usage.md [hunk 4]
```diff
@@ -33,12 +34,20 @@ This component can be used by itself, but is designed to work with Capital Frame
 Theme variables for setting the color and sizes throughout the project.
 Overwrite them in your own project by duplicating the variable `@key: value`.
 
-### Settings
+### Color variables
+
+`$color-` variables are from 18F's
+[US Web Design Standards](https://github.com/18F/web-design-standards/blob/18f-pages/assets/_scss/core/_variables.scss)
```
Comment (Scotchester) [inline src/cf-icons/usage.md]: Update URL and "U.S."

File: src/cf-pagination/src/cf-pagination.less [hunk 5]
```diff
@@ -3,11 +3,34 @@
    Pagination Styling
    ========================================================================== */
 
+//
+// Theme variables
+//
+
+// Color variables
+// $color- variables are from 18F's U.S. Web Design Standards
+// https://github.com/18F/web-design-standards/blob/18f-pages-staging/src/stylesheets/core/_variables.scss
```
Comment (Scotchester) [inline src/cf-pagination/src/cf-pagination.less]: Update URL.

File: src/cf-pagination/usage.md [hunk 6]
```diff
@@ -26,21 +26,18 @@ Theme variables for setting the color and sizes throughout the project. Overwrit
 
 `$color-` variables are from 18F's [US Web Design Standards](https://github.com/18F/web-design-standards/blob/18f-pages/assets/_scss/core/_variables.scss)
```
Comment (Scotchester) [inline src/cf-pagination/usage.md]: Update URL and "U.S."

File: CHANGELOG.md (hunk 1 above)
Comment (jimmynotjim) [inline CHANGELOG.md]: Oh right. I'm also not sure how the `all components` here will shake out with the other PRs we've just merged if they're all released as one. @contolini can you confirm this was the issue with the v4 release or was it something else?

Either way, I also realized I forgot a `[MINOR]` in here

File: src/cf-buttons/usage.md (hunk 2 above)
Comment (jimmynotjim) [inline src/cf-buttons/usage.md]: These were already done in the other PR. I need to rebase now that it's merged.

**Comment by jimmynotjim** [comment]
@Scotchester went ahead and merged the long Changelog entry into one line. Also rebased with Canary to fix the conflicts. Should be good to go once Travis finishes.

**Pull request merged**